mapie/classification.py

```diff
@@ -47,7 +42,7 @@ class MapieClassifier(BaseEstimator, ClassifierMixin):
         Method to choose for prediction interval estimates.
```

Comment:
It would be useful for users to have comment like "warning depreciated use instead conformity_score_function_" comment here

Reply:
Add keyword [DEPRECIATED] Done 👍  (line 71)